internal/pkg/deploy/cloudformation/app.go

```diff
@@ -456,9 +456,12 @@ func (cf CloudFormation) cleanUpRegionalResources(app *config.Application, regio
 	if err := s3.EmptyBucket(resources.S3Bucket); err != nil {
 		return err
 	}
+	ecrRepoName := func(app, workload string) string {
+		return fmt.Sprintf("%s/%s", app, workload)
+	}
 	ecr := cf.regionalECRClient(region)
-	for repo := range resources.RepositoryURLs {
-		if err := ecr.ClearRepository(repo); err != nil {
+	for svcName, _ := range resources.RepositoryURLs {
```

Comment:
rip sorry for the churn :'( static check says my suggestion is stoopid

Reply:
Yeah but `for svcName := range resources.RepositoryURLs {` def. made me churn too...